.github/workflows/check-labels.yml

```
@@ -0,0 +1,31 @@
+name: Check Labels
+
+on:
+  pull_request:
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)